default/scripting/specials/planet/MODERATE_TECH_NATIVES.focs.txt

```diff
@@ -8,6 +8,7 @@ Special
         Planet
         Unowned
         Species
+        # @content_tag{PRIMITIVE} Prevents this special at a location with tag
```

Comment:
clarify: prevents this special from being added to a location with this tag during universe generation